python/lsst/ctrl/orca/StatusListener.py

```diff
@@ -47,20 +48,15 @@ def workflowStarted(self, name):
         """Called when a workflow has started up correctly and is
            ready to process data.
 
-           Notes
-           -----
-           If a pipeline is waiting for an event, the listener should be notified via a workflowWaiting()
-           message.
+        Notes
+        -----
+        If a pipeline is waiting for an request, the listener should be
+        notified via workflowWaiting
         """
         return
 
     def workflowWaiting(self, name):
-        """Indicate that a workflow is waiting for an event to proceed.
+        """Indicate that a workflow is waiting for an request to proceed.
```

Comment:
that class  predates most of those standards by a number of years.  :-)   I'll fix this here;  the whole package is due for a scrub to clean all this up to be sure it's all up to standard for autopep8.